ci/infra/testrunner/tests/test_istio_service_mesh.py

```diff
@@ -0,0 +1,159 @@
+import logging
+import pytest
+import time
+
+CLUSTERROLEBINDING = ("""
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: istio-test
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: suse:caasp:psp:privileged
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: default
+- kind: ServiceAccount
+  name: httpbin
+  namespace: default
+- kind: ServiceAccount
+  name: bookinfo-productpage
+  namespace: default
+- kind: ServiceAccount
+  name: bookinfo-reviews
+  namespace: default
+- kind: ServiceAccount
+  name: bookinfo-ratings 
+  namespace: default
+- kind: ServiceAccount
+  name: bookinfo-details
+  namespace: default
+- kind: ServiceAccount
+  name: sleep
+  namespace: default
+EOF
+""")
+
+
+def _istio_bookinfo_setup(kubectl):
+    istioctl = ("""
+                istioctl --kubeconfig={config} manifest apply \
+                         --set profile=default \
+                         --set addonComponents.prometheus.enabled=false \
+                         --set hub=registry.suse.de/devel/caasp/4.5/containers/containers/caasp/v4.5 \
+                         --set tag=1.5.4 \
```

Comment:
Maybe this version can be in a variable to make the test easier to maintain?

Reply:
yes, makes sense